WISE-Community/WISE
# Added unit plan view back to VLE

To test:
1. Preview a unit.
1. Complete some steps.
1. Click on the "Unit Plan" button on the far right of the step toolbar.
1. Verify that the unit plan view appears and is functional.
1. Verify that completion icons appear for steps that have been completed.
1. Log in as a student and launch a run.
1. Repeat steps 2-5 above.

Resolves #2204.
